schema/partner.js

```diff
@@ -86,6 +91,24 @@ const PartnerType = new GraphQLObjectType({
           }));
         },
       },
+      artworks: {
+        type: new GraphQLList(Artwork.type),
+        args: {
+          size: {
+            type: GraphQLInt,
+          },
+          sort: ArtworkSorts,
+          exclude: {
+            type: new GraphQLList(GraphQLString),
+          },
+        },
+        resolve: ({ id }, options) => {
+          return gravity(`filter/artworks`, assign({}, options, {
```

Comment:
I don't think this is a great idea.

`/filter/artworks` has to make a round trip to an external service (ElasticSearch) as well as hitting Mongo to materialize results. For some functionality, like a complex filter with aggregated counts, that would be very expensive with Mongo (and we don't even support most of that), and so ES is the way to go.

For enumerating a partner's artworks however, I think this will be slower than just using `/partner/:id/artworks`, and feels like overkill. `Partner` and `Artwork` already have a `has_many`/`belongs_to` and so it should be trivial to query them directly thru that endpoint. I'm guessing that any sort that is needed can be trivially added there as well.